### PR TITLE
RW-5: Cleanup block_date in nft old events

### PR DESCRIPTION
**Pull request by 0xRobin**
Cleanup `block_date` in nft old events.
follow up from:
https://github.com/duneanalytics/spellbook/pull/4416

CI:refresh-all-seeds

### CODE REVIEW

**Comment by jeff-dude** [comment]
we can revisit a rerun here once gh actions seem to clear up -- we've had queue delays and longer runtimes today

**Comment by jeff-dude** [comment]
now need to wait for spark in prod to run clean

File: macros/models/_sector/nft/platforms/seaport_v3_trades.sql [hunk 1]
```diff
@@ -558,15 +554,14 @@ with source_ethereum_transactions as (
         ,order_hash
         ,b.fee_wallet_name
   from iv_base_pairs a
-        left join iv_volume b on b.block_date = a.block_date  -- tx_hash and evt_index is PK, but for performance, block_time is included
+        left join iv_volume b on b.block_time = a.block_time  -- tx_hash and evt_index is PK, but for performance, block_time is included
```
[the remark this answers is not in the document]
Comment (jeff-dude) [reply macros/models/_sector/nft/platforms/seaport_v3_trades.sql:557]: i'll wait to see feedback here prior to merging

[the remark this answers is not in the document]
Comment (0xRobin) [reply macros/models/_sector/nft/platforms/seaport_v3_trades.sql:557]: I've used them interchangeably when optimizing join conditions. Does `block_number` perform better @aalan3?

[the remark this answers is not in the document]
Comment (aalan3) [reply macros/models/_sector/nft/platforms/seaport_v3_trades.sql:557]: `block_number` has less distinct values so it should perform a bit better but not sure how big of a difference this makes in practice.

[the remark this answers is not in the document]
Comment (aalan3) [reply macros/models/_sector/nft/platforms/seaport_v3_trades.sql:557]: actually nvm I checked with Jonas and he pointed out that they are going to have the same number of distinct values

**Comment by 0xRobin** [comment]
> spark is still giving us a hard time in prod, but i can look to merge some point today and manually mess with the env there

Note the legacy models on this PR aren't actually modified but are only moved, so a full refresh on them is not really necessary.

**Pull request merged**